src/js/actions/userActions.js

```diff
@@ -62,6 +62,20 @@ export const loginUser = userData => (dispatch, getState) =>
       return Promise.all([Promise.reject(err), dispatch(handleLoginError(err, has2FA))]);
     });
 
+export const verifyTFA = tfaData => dispatch =>
```

Comment:
just for the sake of consistency that could well go as `verify2FA`, also to prevent issues with the potential introduction of a 3 or 10, 12, 13, ... factor methods 😉

Reply:
> just for the sake of consistency that could well go as `verify2FA`, also to prevent issues with the potential introduction of a 3 or 10, 12, 13, ... factor methods 😉

hahah. good! fixed.